geomstats/visualization/hyperbolic.py

```diff
@@ -51,64 +40,42 @@ def convert_to_klein_coordinates(points):
         klein_coords = gs.concatenate([coords_0, coords_1], axis=1)
         return klein_coords
 
-    def draw(self, ax, **kwargs):
-        circle = plt.Circle((0, 0), radius=1.0, color="black", fill=False)
+    def plot_space(self, ax=None, color="black", **kwargs):
+        """Plot points in the KleinDisk"""
+        ax = self.set_ax(ax=ax)
+
+        circle = plt.Circle((0, 0), radius=1.0, color=color, fill=False)
         ax.add_artist(circle)
         points_x = [gs.to_numpy(point[0]) for point in self.points]
         points_y = [gs.to_numpy(point[1]) for point in self.points]
         ax.scatter(points_x, points_y, **kwargs)
```

Comment:
`plot_space` should be responsible to plot only "the space", without points. Here usually we mean some wireframe style plot.

`plot` receives `points` (check in `Plotter`) and is where points can actually be plotted.